tendrl/commons/objects/cluster_alert/__init__.py

```diff
@@ -12,7 +13,13 @@ def __init__(self, *args, **kwargs):
         self.value = 'alerting/clusters/{0}/{1}'
 
     def render(self):
-        self.value = self.value.format(self.tags['integration_id'],
-                                       self.alert_id
-                                       )
+        try:
+            self.tags = json.loads(self.tags)
+        except TypeError:
```

Comment:
json.loads raises ValueError and not a TypeError, why have you used TypeError?

Reply:
@r0h4n For none value and if value already json then it will raise TypeError also , it will add both